app/src/command.c

```diff
@@ -18,9 +22,30 @@ static inline const char *get_adb_command() {
     return adb_command;
 }
 
+static void show_err_msg(int err) {
+#ifdef __WINDOWS__
+    LOGE("Failed to execute adb.\n");
```

Comment:
Mostly windows users are lost with `adb`, isn't it possible to know when the problem is a missing adb?